source/user_guide.rst

```diff
@@ -0,0 +1,93 @@
+`OBS <https://build.opensuse.org/>`_. It allows packagers to maintain their
+package sources in the `pool <https://src.opensuse.org/pool>`_ organization
+instead of on OBS. Submissions are handled via pull requests on
+https://src.opensuse.org instead of submit requests on OBS.
+
+A bot has been setup that will automatically forward eligible pull requests as
+submitrequests directly to openSUSE:Factory.
+
+
+Packager Guide
+--------------
+
+.. hint::
+   Consider adjusting the package description to alert contributors that
+   ordinary submitrequests are no longer possible and where they should send a
+   pull request instead.
+
+To participate in the test drive, modify the development package in the
+developroject as follows:
+
+1. Login to https://src.opensuse.org using the openSUSE IDP provider.
+
+2. Clone your package from https://src.opensuse.org/pool/$pkg_name and switch to
+   the ``factory`` branch.
+
+3. Checkout your package from the configured develproject from OBS:
+
+.. code-block:: bash
+
+   osc co $(osc develproject openSUSE:Factory $pkgname)
+
+4. Compare your package in git (from step 2) with the package on OBS (from step
+   3). If there are differences, fork the package on src.opensuse.org and send a
+   pull request.
+
+5. Modify the package meta of the devel package via :command:`osc meta -e pkg
+   $develprj $pkgname` by adding the following entry, which will automatically
+   synchronize your package from git to OBS.
+
+.. code-block:: xml
+
+   <package name="$pkg_name" project="$develprj">
+     <!-- snip -->
+     <scmsync>https://src.opensuse.org/pool/$pkg_name#factory</scmsync>
+   </package>
+
+6. [Recommended] setup branch protections for the `factory` branch and require
+   pull requests. Direct pushes to the `factory` branch will **not** be
+   forwarded by ``scm-staging-bot`` (however, ``factory-auto`` will
+   eventually submit them).
+
+7. Pull requests against the ``factory`` branch by a package or project
+   maintainer on OBS will automatically get forwarded as submitrequests. Pull
+   requests from all other users have to be approved by a maintainer first,
+   before a submit request to ``openSUSE:Factory`` is created.
+
+   Note that the ``factory-auto`` bot will decline more than one submission to
+   the same package in ``openSUSE:Factory``. You can thus use pull request
+   approvals as a selection which pull request to send first.
+
+
+Contributor Guide
+-----------------
+
+
+1. Find the package that you want to contribute to in src.opensuse.org/pool and
+   fork it from the repository src.opensuse.org/pool/$pkg_name
+
+2. Clone your fork via :command:`git clone src.opensuse.org/$username/$pkg_name`
+   and implement your changes:
+
+.. code-block:: shell-session
+
+   cd $pkg_name
+   $EDITOR $pkg_name.spec
+   osc vc -m "$CLEVER_CHANGELOG_ENTRY" $pkg_name.changes
+
+3. [optional] Run a local build via :command:`osc build openSUSE:Factory
```

Comment:
this step is only needed if the package isn't checked out via osc. if you're maintainer in the devel project package, you can simply add a `git add remote my gitea@src.opensuse.org/path/to/package` and commit then locally via git and push to "my", and create a PR that way.

Reply:
The guide doesn't mention that the package can be checked out via `osc`, hence I wouldn't add this part to not confuse the reader more.

Reply:
even if it simplifies the instructions?